bump_version.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+#
+# Description: Generate the next release version
+
+set -uo pipefail
```

Comment:
```suggestion
set -uneo pipefail
```

Reply:
Nooooooope, I do not `set -e`. No silent exits.